screens/OnboardingScreen/index.tsx

```diff
@@ -24,7 +24,7 @@ export const Day1Screens = [
   <Intro5 key={87922} />,
 ];
 export const Day2And3Screens = (recordingDay: number) => [
-  <Intro1Day2 key={22} recordingDay={recordingDay} />,
+  <Intro1Day2 key={22} recordingDay={recordingDay} setCanScroll={(c) => {}} />,
```

Comment:
Oh my I forgot about this one. Oops! I think a way to do this a bit cleaner is to pass in a Boolean to the IntroScreen component that says whether it can scroll or not. Either way works tho.

Reply:
good point. since it's just a fixing types issue, I'm going to just make it an optional prop